batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java

```diff
@@ -58,6 +58,8 @@ public class CreateResourcesIT {
           + UUID.randomUUID().toString().substring(0, 7);
   private static final String CUSTOM_EVENT_NAME = "test-job"
           + UUID.randomUUID().toString().substring(0, 7);
+  private static final String JOB_ALLOCATION_POLICY_LABEL = "test-job"
```

Comment:
```suggestion
  private static final String JOB_ALLOCATION_POLICY_LABEL = "test-job-allocation-label"
```

Always have a unique suffix to every job name to denote the job.